static/include/how-to/gather-sync-sensor.md

```diff
@@ -0,0 +1,48 @@
+Viam's [data management service](/services/data/) lets you capture data locally from sensors and then sync it to the cloud where you can access all data across different {{< glossary_tooltip term_id="machine" text="machines" >}} or {{< glossary_tooltip term_id="location" text="locations" >}} and see trends over time.
```

Comment:
"and see trends over time" feels like a bit much since the app itself doesn't have built-in data visualization tools and it'd be somewhat hard to see trends over time just by looking at raw data. I think it would make it easier to read if this sentence was a little shorter anyway:
```suggestion
Viam's [data management service](/services/data/) lets you capture data locally from sensors and then sync it to the cloud where you can access all data across different {{< glossary_tooltip term_id="machine" text="machines" >}} or {{< glossary_tooltip term_id="location" text="locations" >}}.
```